pkg/rc/farm.go

```diff
@@ -228,7 +229,7 @@ START_LOOP:
 				go func(id fields.ID) {
 					defer func() {
 						if r := recover(); r != nil {
-							err := util.Errorf("Caught panic in rc farm: %s", r)
+							err := util.Errorf("Caught panic in rc farm: %s\n%s", r, debug.Stack())
```

Comment:
maybe I should just be logging `err.Stack()` in the log message below, which is what I really care about